src/identity/manager.rs

```diff
@@ -287,6 +293,14 @@ impl Worker {
                     },
                     None => break 'main,
                 },
+
+                // Update backoff to have values more appropriate for certificate refresh failures.
+                backoff.initial_interval = CERT_REFRESH_FAILURE_RETRY_DELAY_INITIAL_INTERVAL; // start with 500ms
+                backoff.max_interval = CERT_REFRESH_FAILURE_RETRY_DELAY_MAX_INTERVAL; // max out at 60s
+                backoff.multiplier = 2.0; // double the interval each time
+                backoff.randomization_factor = 0.2; // 20% randomization
+                backoff.max_elapsed_time = Some(Duration::from_secs(300)); // max out at 5 minutes
```

Comment:
I think this is broken. After 5min it returns None and therefor CERT_REFRESH_FAILURE_RETRY_DELAY_INITIAL_INTERVAL -- not 5 minutes. 

Experienced learned from outage in the Go equivilent https://github.com/istio/istio/blob/50fe4b86425bd106810d323280dd5f8098fb082b/pkg/backoff/exponential.go#L77 :slightly_smiling_face:

Reply:
Ahh yes, good catch!